src/index.html.ejs

```diff
@@ -6,7 +6,6 @@
   <meta name="description" content="Stuff" />
   <link rel="shortcut icon" href="<%= htmlWebpackPlugin.files.favicon %>">
 
-  <!-- FIXME is this a CDN? Maybe fetch from someplace else  -->
```

Comment:
This IS a CDN. Alternatively we might want to bundle it with our CSS, but :man_shrugging: